docs/features.md

```diff
@@ -52,6 +52,16 @@ The `Disruption` resource uses [label selectors](https://kubernetes.io/docs/conc
 
 **NOTE:** If you are targeting pods, the disruption must be created in the same namespace as the targeted pods.
 
+### DynamicTargeting
+
+By default, targeting is limited to a single target selection at the disruption's creation. It allows for more controlled disruption impact and propagation, as the targets will never change and canbe compensated in case of issue. Its major limit is not being able to follow targets through deployments/rollouts.
+By setting the `DynamicTargeting` flag to `true` in the Disruption's yaml spec, you activate a constant re-targeting for this disruption. This means at any given time, any target within the selector's scope will be added to the target list and be disrupted.
+This is a feature to use with care, as it can quickly get out of control: per example, a disruption targeting 100% of an application's pod will affect all existing **and** future pods which can appear once the disruption started. As long as this 100% disruption exists, there will be no spared pod.
```

Comment:
Let's discuss this within the team, as I personally think we may want to default to this being true in the future, as a lot of users seem to expect this behavior

Reply:
I'll start a thread !

Reply:
Follow-Up: decided to have a "StaticTargeting" flag defaulted to true as a soft transition
Default behaviour will be in time become StaticTargeting set to false